### DISCUSSION TITLE
[Feature Request] Breadcrumb

### DISCUSSION THREAD

**Discussion by calvo-jp** [cschroeter/park-ui#399] [Ideas]
[breadcrumb](https://chakra-ui.com/docs/components/breadcrumb) component similar to chakra ui. TIA

**Comment by Horsty80** [comment]
Hello @cschroeter why this feature request is closed ? Breadcrumb are available through Ark here: https://ark-ui.com/react/docs/components/steps

It's will be a good improvement for ParkUi to add it :)

Thanks

**Comment by gabrielgrant** [comment]
[Breadcrumbs are listed as an upcoming Park UI feature on the Ark UI feature tracker](https://ark-ui.canny.io/feature-requests/p/breadcrumb). Is there some further discussion to be had about this, or is it just that you prefer future features be kept as discussions rather than issues @cschroeter ?
